From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: 5656565566 <contact@home56.top>
Date: Sun, 16 Aug 2026 22:49:12 +0800
Subject: [PATCH] Allow Floral NativeBridge private headers in updatable modules

The ART mainline module consumes the internal Floral NativeBridge header
library used by compatibility loader routing. Add the header library to the
Android 12 APEX dependency allowlist.

---
 build/allowed_deps.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/allowed_deps.txt b/build/allowed_deps.txt
index ad589700a3d1..15faa899757a 100644
--- a/build/allowed_deps.txt
+++ b/build/allowed_deps.txt
@@ -374,6 +374,7 @@ libmpeg2dec(minSdkVersion:29)
 libnanohttpd(minSdkVersion:30)
 libnativebase_headers(minSdkVersion:29)
 libnativebridge-headers(minSdkVersion:S)
+libnativebridge-private-headers(minSdkVersion:S)
 libnativehelper_compat_libc++(minSdkVersion:(no version))
 libnativehelper_compat_libc++(minSdkVersion:29)
 libnativehelper_header_only(minSdkVersion:29)
-- 
2.47.3

